src/translations/en.json

```diff
@@ -1464,7 +1464,7 @@
           "caption": "Home Assistant Cloud",
           "description_login": "Logged in as {email}",
           "description_not_login": "Not logged in",
-          "description_features": "Control away from home, integrate with Alexa and Google Assistant.",
+          "description_features": "Control away from home; integrate with Alexa and Google Assistant",
```

Comment:
The two items are unrelated, I think. Also I dropped the period since no other description had one.

Reply:
Might be a personal thing, but to me ";" looks weird here plus I would switch the first part.

Perhaps: "Control home when away + integrate with Alexa and Google Assistant"

Reply:
> Control away from home and integrate with Alexa and Google Assistant
?

Reply:
I would vote for having the word "when" somewhere in there. 

=> Control home when away and integrate with Alexa and Google Assistant